lib/KLogo/__tests__/KLogo.spec.js

```diff
@@ -0,0 +1,62 @@
+import { renderComponentForVisualTest, takeSnapshot } from '../../../jest.conf/visual.testUtils';
+
+describe.visual('KLogo', () => {
+  const snapshotOptions = { widths: [400], minHeight: 512 };
+  describe('Render KLogo with different props', () => {
+    it('Render a default logo', async () => {
+      await renderComponentForVisualTest('KLogo', { altText: 'Kolibri default logo', size: '150' });
+      await takeSnapshot('KLogo - default', snapshotOptions);
+    });
+    it('Render a logo with background', async () => {
+      await renderComponentForVisualTest('KLogo', {
+        altText: 'Kolibri Logo with background',
+        size: '150',
+        showBackground: true,
+      });
+      await takeSnapshot('KLogo - background', snapshotOptions);
+    });
+    it('Render a logo with rectangular background', async () => {
+      await renderComponentForVisualTest('KLogo', {
+        altText: 'Kolibri rectangular background',
+        size: '150',
+        backgroundStyle: 'rect',
+        showBackground: true,
+      });
+      await takeSnapshot('KLogo - rectangular background', snapshotOptions);
+    });
+    it('Render a logo with dimensions', async () => {
```

Comment:
Can you change dimensions in this test to make it observable whether setting dimensions works or no? Right now, in Percy output I cannot tell the difference between the default state and the state with dimensions set. For example, you could make the logo twice its default size.

And along the lines of my other comments, I recommend cleaning up unnecessary props and really just focus on those that are needed for this particular test. Here we are testing dimensions, so there's no need to tweak background etc. This makes maintenance and clarity of tests better.